ci-operator/step-registry/baremetal/lab/upi/conf/network/vips/ipv4/baremetal-lab-upi-conf-network-vips-ipv4-commands.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o errtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# shellcheck disable=SC2154
+if [ "${AGENT_PLATFORM_TYPE}" = "none" ] || [ "${masters}" -eq 1 ]; then
```

Comment:
```suggestion
if [ "${AGENT_PLATFORM_TYPE}" = "none" ]; then
```

We don't need this that other condition anymore, right?

Reply:
Discussed in call: still needed.